addon/components/flexberry-lookup.js

```diff
@@ -286,21 +308,58 @@ export default FlexberryBaseComponent.extend({
     this._super();
   },
 
+  didDestroyElement() {
+    this._super();
+
+    this.removeObserver('i18n.locale', this, this._languageReinit);
+  },
+
   // Init component when DOM is ready.
   didInsertElement: function() {
     this._super();
+    this.addObserver('i18n.locale', this, this._languageReinit);
+  },
 
-    if (this.get('readonly')) {
-      return;
-    }
+  /**
+    Handles changing current locale.
+    It reinits autocomplete or dropdown mode (depending on flag) in order to localize messages.
 
+    @private
```

Comment:
Check please, must be like this:
/**
@method _toString
@private
*/
and
/**
@property _name
@private
*/